src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotJVMCIRuntime.java

```diff
@@ -926,22 +926,16 @@ JavaType lookupTypeInternal(String name, HotSpotResolvedObjectType accessingType
 
     /**
      * Gets the {@code jobject} value wrapped by {@code peerObject}. The returned "naked" value is
```

Comment:
Remove `"naked"` since a JNI value is a proper handle and not really naked.

Reply:
Fixed by d10f81847f27dfe413ace2521aec2f76647f4071